--- libltdl/configure.ac	Wed Feb 20 21:09:31 2002
+++ libltdl/configure.ac	Wed Feb 20 21:10:52 2002
@@ -32,12 +32,7 @@
 ## Libltdl specific configuration. ##
 ## ------------------------------- ##
 
-# We shouldn't be using these internal macros of autoconf,
-# but CONFIG_AUX_DIR($with_auxdir) breaks automake.
-AC_ARG_WITH([auxdir],
-    [AC_HELP_STRING([--with-auxdir=DIR], [path to autoconf auxiliary files])],
-    [AC_CONFIG_AUX_DIRS($with_auxdir)],
-    [AC_CONFIG_AUX_DIR_DEFAULT])
+AC_CONFIG_AUX_DIR(../tools)
 
 if test -z "$enable_ltdl_install$enable_ltdl_convenience"; then
   if test -f ${srcdir}/ltmain.sh; then
